manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -407,6 +408,62 @@ private function addDefaultDoctrineMapping(array $extensionConfigs, ContainerBui
         return $extensionConfigs;
     }
 
+    /**
+     * Enable SQL strict mode for PDO and mysql drivers by setting an init
+     * command unless already set.
+     *
+     * @return array<string,array<string,array<string,array<string,mixed>>>>
+     */
+    private function enableStrictMode(array $extensionConfigs, ContainerBuilder $container): array
+    {
+        $driver = '';
+        $url = null;
+        $options = [];
+
+        foreach ($extensionConfigs as $config) {
+            if (null !== ($driverConfig = $config['dbal']['connections']['default']['driver'] ?? null)) {
+                $driver = $driverConfig;
+            }
+
+            if (null !== ($urlConfig = $config['dbal']['connections']['default']['url'] ?? null)) {
+                $url = $container->resolveEnvPlaceholders($urlConfig, true);
+            }
+
+            if (null !== ($optionsConfig = $config['dbal']['connections']['default']['options'] ?? null)) {
+                $options = array_replace($options, $optionsConfig);
+            }
+        }
+
+        if (null !== $url) {
+            $driver = str_replace('-', '_', parse_url($url, PHP_URL_SCHEME));
+        }
+
+        // Skip if driver is not supported
+        if (null === ($key = ['pdo_mysql' => 1002, 'mysqli' => 3][$driver] ?? null)) {
```

Comment:
```suggestion
        if (null === ($key = ['pdo_mysql' => 1002, 'mysql' => 1002, 'mysql2' => 1002, 'mysqli' => 3][$driver] ?? null)) {
```

`mysql` and `mysql2` are aliases for `pdo_mysql`

Reply:
see c0f041cd97dc5ab54352c470f1dd1a918c1d3344